src/source-dpdk.c

```diff
@@ -152,12 +152,18 @@ static void DPDKFreeMbufArray(struct rte_mbuf **mbuf_array, uint16_t mbuf_cnt, u
 static uint64_t CyclesToMicroseconds(const uint64_t cycles)
 {
     const uint64_t ticks_per_us = rte_get_tsc_hz() / 1000000;
+    if (ticks_per_us == 0) {
```

Comment:
@lukashino is this necessary? Or can rte_get_tsc_hz return < 1000000?

Reply:
Seems to be possible only when:
- the CPU would have a clock rate less than 100MHz
- the CPU would not be well supported by DPDK and some register would return 0/random number instead ticks per second

So, assuming every CPU has a clock rate of at least 800MHz, and we will not care about unsupported CPUs then it should be fine even without the check.